### DISCUSSION TITLE
A tool to archive content

### DISCUSSION THREAD

**Discussion by peterbe** [mdn/yari#2207] [Ideas]
Look at https://github.com/mdn/content/blob/main/files/en-us/mozilla/developer_guide/build_instructions/old_thunderbird_build/index.html for example. It's about how to compile Thunderbird for versions older than version 59. (2 years ago). Definitely not very "web dev" content. (Besides a bunch of Thunderbird content got put into the github.com/mdn/archived-content repo in the migration). Not the point.

The point is that perhaps it would be useful for the content contributors to be able to migrate individual documents, or even better, individual sub-trees *out* from `mdn/content` and into `mdn/archived-content`. 
It's totally possible to do today. But messy. You copy-n-paste the rendered HTML from developer.mozilla.org,  do `git rm -r ...`  in the mdn/content repo, and then do a `git add` in the mdn/archived-content repo by cobbling together the front-matter at the top of the `index.html` files. 

If we develop a (CLI) tool to make it really easy, power users, can you just type something like this:

```bash
cd /path/to/mdn/content
export CONTENT_ARCHIVED_ROOT=/path/to/mdn/archive-content/files
yarn archive files/en-us/mozilla/obscure/old
```
and then all you need to do is to type whatever you do to push your git changes into the two repos. 

Being able to reduce the size of the content repo is useful for many reasons.

**Comment by chrisdavidmills** [comment]
This totally sounds useful. Yes!

**Comment by schalkneethling** [comment]
@mdn/core-dev Does this sound like a discussion we want to keep around and bring to more people's attention, or should we remove the discussion. There are unfortunately only two options, lock or delete. Thank you.

**Reply by caugner** [reply]
My best guess is that nowadays archiving is necessary only very rarely, especially since the whole `/mozilla` section has been archived. So it does not seem to be justified investing in that tool.

I would opt for locking the discussion.

**Reply by schalkneethling** [reply]
Thank you, @caugner. If there is consensus on this, please feel free to lock the discussion.
